setup.cfg

```diff
@@ -2,3 +2,36 @@
 python_files=*.py
 python_classes=_UnitTest
 python_functions=_unittest
+
+[mypy]
+# Python version is not specified to allow checking against different versions
+warn_return_any        = True
+warn_unused_configs    = True
+disallow_untyped_defs  = True
+check_untyped_defs     = True
+no_implicit_optional   = True
+warn_redundant_casts   = True
+warn_unused_ignores    = True
+show_error_context     = True
+
+[mypy-pytest]
+ignore_missing_imports = True
+
+[mypy-pydsdl]
+ignore_missing_imports = True
+
+[mypy-nunavut]
+ignore_missing_imports = True
+
+[mypy-nunavut.*]
+ignore_missing_imports = True
+
+[mypy-numpy.*]
+ignore_missing_imports = True
```

Comment:
This program does not use numpy, nunavut, or pydsdl; what's the point of listing them here?